crates/networking/rpc/engine/fork_choice.rs

```diff
@@ -302,17 +303,17 @@ fn handle_forkchoice(
                     context
                         .storage
                         .update_sync_status(false)
+                        .await
                         .map_err(|e| RpcErr::Internal(e.to_string()))?;
                     let current_head = context.storage.get_latest_canonical_block_hash()?.ok_or(
                         RpcErr::Internal("Missing latest canonical block".to_owned()),
                     )?;
                     let sync_head = fork_choice_state.head_block_hash;
-                    tokio::spawn(async move {
-                        // If we can't get hold of the syncer, then it means that there is an active sync in process
```

Comment:
I need to restore the comment. It was lost during a period where this task would break the build.